data-prepper-plugins/s3-source/build.gradle

```diff
@@ -13,6 +13,10 @@ repositories {
 
 dependencies {
     implementation project(':data-prepper-api')
+    implementation project(':data-prepper-plugins:parquet-codecs')
```

Comment:
This project should not include these codecs. The integration test implementation should have them for testing. But, not in `implementation`.

Reply:
S3 source has `integrationTest` folder inside `src` folder that has `CsvRecordsGenerator`, `JsonRecordGenerator` and `NewLineDelimitedRecordsGenerator` classes require `CsvInputCodec`,  `JsonInputCodec` and `NewlineDelimetedInputCodec` classes respectively and if we put them in testImplementation than mentioned classes would not accessible inside `s3-Source.src.integrationTest`.

Do we need to use only old s3 codec implementation inside `integrationTest`?